frontend/src/components/Scheduling/SmartSchedulingDashboard.jsx

```
@@ -0,0 +1,489 @@
+
+  const [analytics, setAnalytics] = useState(null);
+  const [appointments, setAppointments] = useState([]);
+  const [insights, setInsights] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [activeTab, setActiveTab] = useState("overview");
+
+  useEffect(() => {
+    if (user) {
+      loadDashboardData();
+    }
+  }, [user, loadDashboardData]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)